pkg/api/mlflow/service/artifact/validators.go

```diff
@@ -8,8 +8,8 @@ import (
 	"github.com/G-Research/fasttrackml/pkg/api/mlflow/api/request"
 )
 
-// RelativePathRegExp checks for the sequence `./` or `../` in provided path.
-var RelativePathRegExp = regexp.MustCompile(`\.{2}\/?`)
+// RelativePathRegExp checks for the `../` in provided path.
+var RelativePathRegExp = regexp.MustCompile(`\.{2}\/`)
```

Comment:
This is letting `..` go through (without `/` at the end). Please add a few unit tests like these:
- ok
  - `foo`
  - `./foo`
  - `./foo/`
  - `.foo`
  - `foo.bar`
  - `foo..bar`
- error
  - `..`
  - `foo/../bar`